eo-maven-plugin/src/test/java/org/eolang/maven/ProbeMojoTest.java

```diff
@@ -205,15 +207,15 @@ void findsProbesWithVersionsInDifferentObjectionaries(@TempDir final Path temp)
                 "Tojos should have contained versioned object %s after probing, but they didn't",
                 ProbeMojoTest.STDOUT
             ),
-            maven.externalTojos().contains(ProbeMojoTest.STDOUT),
+            maven.externalTojos().contains(ProbeMojoTest.STDOUT.toString()),
             Matchers.is(true)
         );
         MatcherAssert.assertThat(
             String.format(
                 "Tojos should have contained versioned object %s after probing, but they didn't",
-                number
+                text
             ),
-            maven.externalTojos().contains(number),
+            maven.externalTojos().contains(text.toString()),
```

Comment:
@maxonfjvipon What do you think if we pass `ObjectName` directly to `ForeignTojos.contatins` method:
```java
maven.externalTojos().contains(text)
```
It could make the code free from the frequent `test.toString()` invocations.